Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -623,6 +667,21 @@ private void SetHandDataFromController(IMixedRealityController controller, bool
                 }
             }
 
+            //store value in case of MRController
+            if (controller.InputSource.Pointers.Length > 0 )
```

Comment:
Again it's dangerous to assume that GGVPointer is the first pointer in the list. Instead, you can check if the controller is a WindowsMixedRealityGGVHand

Reply:
will do